src/Home.vue

```diff
@@ -22,117 +22,22 @@
 
 <template>
 	<NcContent id="content" app-name="workspace">
-		<notifications
-			position="top center"
-			width="50%"
-			class="notifications"
-			:close-on-click="true" />
-		<NcAppNavigation v-if="$root.$data.canAccessApp === 'true'">
-			<NcAppNavigationNewItem v-if="$root.$data.isUserGeneralAdmin === 'true'"
-				icon="icon-add"
-				:title="t('workspace', 'New space')"
-				@new-item="createSpace" />
-			<NcAppNavigationItem
-				:title="t('workspace', 'All spaces')"
-				:to="{path: '/'}"
-				:class="$route.path === '/' ? 'space-selected' : 'all-spaces'" />
-			<template #list>
-				<NcAppNavigationItem
-					v-for="(space, spaceName) in $store.state.spaces"
-					:key="space.id"
-					:class="$route.params.space === spaceName ? 'space-selected' : ''"
-					:allow-collapse="true"
-					:open="$route.params.space === spaceName"
-					:title="spaceName"
-					:to="{path: `/workspace/${spaceName}`}">
-					<NcAppNavigationIconBullet slot="icon" :color="space.color" />
-					<CounterBubble slot="counter" class="user-counter">
-						{{ $store.getters.spaceUserCount(spaceName) }}
-					</CounterBubble>
-					<div>
-						<NcAppNavigationItem
-							v-for="group in sortedGroups(Object.values(space.groups), spaceName)"
-							:key="group.gid"
-							icon="icon-group"
-							:to="{path: `/group/${spaceName}/${group.gid}`}"
-							:title="group.displayName">
-							<CounterBubble slot="counter" class="user-counter">
-								{{ $store.getters.groupUserCount( spaceName, group.gid) }}
-							</CounterBubble>
-						</NcAppNavigationItem>
-					</div>
-				</NcAppNavigationItem>
-				<!-- <div id="app-settings">
-					<div id="app-settings-header">
-						<button v-if="$root.$data.isUserGeneralAdmin === 'true'"
-							icon="icon-settings-dark"
-							class="settings-button"
-							data-apps-slide-toggle="#app-settings-content">
-							{{ t('workspace', 'Settings') }}
-						</button>
-					</div>
-					<div id="app-settings-content">
-						<NcActionButton v-if="$root.$data.isUserGeneralAdmin === 'true'"
-							:close-after-click="true"
```

Comment:
@acdmft is it possible to rewrite this piece of code with the comment ? Or the linter said you something ?